pkg/builder/volbuilder/build.go

```diff
@@ -21,6 +21,9 @@ import (
 	apis "github.com/openebs/zfs-localpv/pkg/apis/openebs.io/zfs/v1"
 )
 
+// MarkForDeletionAnnotation is the annotation key
+const MarkForDeletionAnnotation string = "openebs.io/marked-for-deletion"
```

Comment:
Shouldn't we use the one from, zfs/volume.go?

Reply:
done